Library/Formula/boot.rb

```diff
@@ -0,0 +1,18 @@
+require "formula"
+
+class Boot < Formula
+  homepage "http://boot-clj.com"
+  version "2.0.0-rc4"
```

Comment:
It could go into `homebrew-devel-only` before final but I think it's probably most sensible just to wait for the final version.

Should this have a `maven` dependency if it needs it? Where is the `jar` file; as far as I can see this is installing a single script?